ext/fg/js/frontend-initialize.js

```diff
@@ -88,8 +95,8 @@ async function main() {
         }
 
         let popup;
-        if (isIframe && options.general.showIframePopupsInRootFrame) {
-            popup = popups.iframe || await createIframePopupProxy(url, frameOffsetForwarder);
+        if (isIframe && options.general.showIframePopupsInRootFrame && !document.fullscreen && iframePopupsInRootFrameAvailable) {
```

Comment:
`document.fullscreen` "is no longer recommended](https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreen). Would `Popup._getFullscreenElement`'s implementation work?

Reply:
https://github.com/FooSoft/yomichan/pull/460/commits/1f071df50e0363a7e52295512a1019d99c4c8434